test/s3select_test.cpp

```diff
@@ -2649,11 +2659,11 @@ TEST(TestS3selectFunctions, presto_syntax_alignments)
 
   auto s3select_res = run_s3select(input_query, input);
 
-  const std::string input_presto_query = "Select t._1,t._2 fRom s3OBJECT t whEre _1 = _2";
+  //const std::string input_presto_query = "Select t._1,t._2 fRom s3OBJECT t whEre _1 = _2";
```

Comment:
this query verifies some syntax definitions which are critical for integration with presto (could be also other utilities).

in this case, it's possible to skip the JSON verification.
some modifications to the syntax parser are required.